newIDE/app/src/UI/SemiControlledTextField.js

```diff
@@ -87,6 +93,22 @@ export default class SemiControlledTextField extends React.Component<
     if (this._field) return this._field.getInputNode();
   }
 
+  _onWheel(evt) {
+    const { focused, text } = this.state;
+
+    if (!focused) return;
+    if (typeof text === 'number') {
```

Comment:
Can be simplified to 
```suggestion
    if (focused && typeof text === 'number') {
```